docs/walkthrough/create-training-data.md

```diff
@@ -62,10 +58,27 @@ At the model saving time, you will discover, we are saving two models to your lo
 ````
 
 
+````{tab} two elements per row
+If you want two elements to be semantically close together, they can be placed on the same row as a pair, doing so will assign each pair a distinct label:
```

Comment:
"If you want two elements to be semantically close together, they can be placed on the same row as a pair. Each pair will automatically be assigned a distinct label:"